eng/testing/tests.singlefile.targets

```diff
@@ -136,6 +135,15 @@
           StandardOutputImportance="Low">
       <Output TaskParameter="ConsoleOutput" PropertyName="CppLinker" />
     </Exec>
+
+    <Exec Command="&quot;$(CppLinker)&quot; -fuse-ld=lld -Wl,--version" Condition="$(CppCompilerAndLinker.Contains('clang'))"
```

Comment:
The logic in `init-compiler.sh` that decides whether we want to use ldd is more complicated.

Would it be better to depend on `LDFLAGS` variable produced by `init-compiler.sh` instead of duplicating the check so that we get the same answer?